product_docs/docs/edbcloud/beta/getting_started/01_understanding_qotas_in_azure.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: Understanding quotas in Azure
+Redirects:
+ - 01_check_resource_limits/e3
+---
+
+The following sections help you plan for quotas (service limits) and Azure Service Provider registrations required by your cluster.
```

Comment:
It seems like this page should start with some preface comments about why you should care about these planning activities. E.g., "BigAnimal is deployed inside your Azure account and so requires you to perform some Azure configuration", something like that, before diving in.

Reply:
Changed the introduction.